src/tests/JIT/interpreter/Interpreter.csproj

```diff
@@ -8,4 +8,7 @@
   <ItemGroup>
     <Compile Include="Interpreter.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <CMakeProjectReference Include="CMakeLists.txt" />
+  </ItemGroup>
```

Comment:
Maybe this is a stupid idea, but what happens if you add this reference to both `Interpreter` and `InterpreterTester` projects ? 😄 . Would this solve the problem by having the native library in both places ?

Reply:
That's a nice compromise that would remove the need for the cwd stuff. I'll try it.